lms/templates/bookmarks/bookmarks_results.underscore

```diff
@@ -0,0 +1,42 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header" tabindex="0"><%= gettext("My Bookmarks") %></h2>
+
+<% if (bookmarks.length) { %>
+
+    <div class='bookmarks-results-list'>
+        <% _.each(bookmarks, function(bookmark, index) { %>
+             <a class="bookmarks-results-list-item" href="<%= bookmark.get('usage_id') %>" aria-describedby="bookmark-type-<%= index %> bookmark-date-<%= index %>">
```

Comment:
@cptvitamin I am using the default mac os screen reader. 
I have updated the sandbox with the changes below:
1. Removed `tabindex=0` from `My Bookmarks` header
2. Added fourth bookmarked link/item when you click on Bookmarks button, this fourth bookmarked item is based on what you suggested in the mockup. On my side screen reader twice reads `FIRST > SECOND > THIRD`